README.md

```diff
@@ -22,11 +22,9 @@ Keyvi - the short form for "Key value index" - defines a special subtype of the
 
 ### Quick
 
-Precompiled binaries for Ubuntu 14.04LTS are available at the [keyvi PPA](https://launchpad.net/~hendrik-muhs/+archive/ubuntu/keyvi). To install use: 
+Precompiled binary wheels are available for OS X and Linux on [PyPi](https://pypi.python.org/pypi/pykeyvi). To install use:
 
-    sudo add-apt-repository ppa:hendrik-muhs/keyvi
-    sudo apt-get update
-    sudo apt-get install keyvi python-keyvi
+    pip install pykeyvi
 
 ### From Source
 
```

Comment:
We should somewhere explain a bit more the relation between the C++ part and the python part. 

As well we should mention python support: 2.7, 3.x, pypy (works but no binary package).